### PR TITLE
Updating entry for Coin Cafe

### PR DESCRIPTION
**Pull request by coincafeinc**

### CODE REVIEW

**Comment by mxxcon** [comment]
if you do support tfa, `twitter` field is not necessary.

File: _data/bitcoin.yml [hunk 1]
```diff
@@ -122,11 +122,14 @@ websites:
...
 
-    - name: Coincafe
+    - name: Coin Cafe
       url: https://coincafe.com
       img: coincafe.png
-      tfa: No
-      twitter: getcoincafe
+      tfa: Yes
+      software: Yes
+      email: Yes
```
Comment (Carlgo11) [inline _data/bitcoin.yml]: The blog post says nothing about email TFA.

**Comment by mxxcon** [comment]
@Carlgo11 actually it does.

> We previously enforced 2FA by email, but now, you can choose to use a software token when you log in or perform sensitive actions with your account, like transferring bitcoins.

**Comment by Carlgo11** [comment]
Ah okay. My bad.

**Comment by coincafeinc** [comment]
OK thanks, we've removed the twitter handle.

**Comment by fpigerre** [comment]
Thanks for the pull request @coincafeinc! :+1:

**Pull request merged**